sunpycube/cube/cube_utils.py

```diff
@@ -613,10 +633,10 @@ def index_sequence_as_cube(cubesequence, item):
                                  cubesequence.common_axis, len(cubesequence[0].shape)))
         sequence_index, cube_index = _convert_cube_like_slice_to_sequence_slices(
             item_list[cubesequence.common_axis], cumul_cube_lengths)
-        if sequence_index.start != sequence_index.stop:
-            raise ValueError("Slicing across multiple cubes not yet supported.")
-        else:
-            sequence_index = sequence_index.start
+        # if sequence_index.start != sequence_index.stop:
+        #     raise ValueError("Slicing across multiple cubes not yet supported.")
+        # else:
+        #     sequence_index = sequence_index.start
```

Comment:
Delete these commented out lines as slicing across cubes should now be supported.